src/bivariate_normals.jl

```diff
@@ -326,7 +326,7 @@ Note that nubar is not included.
 """
 function GalaxySigmaDerivs{NumType <: Number}(
     e_angle::NumType, e_axis::NumType, e_scale::NumType,
-    XiXi::SMatrix{2,2,NumType,4}, nuBar::NumType = one(NumType), calculate_tensor::Bool=true)
+    XiXi::SMatrix{2,2,NumType,4}, nuBar::Float64=1.0, calculate_tensor::Bool=true)
```

Comment:
> Did you have a reason to think otherwise?

No. I was just curious what was causing the allocation and it wasn't obvious to me. I didn't think that it should matter if you used `one` or `1.0` here.